modules/data.atmosphere/R/upscale_met.R

```diff
@@ -1,87 +1,85 @@
-substrRight <- function(x, n) {
-  substr(x, nchar(x) - n + 1, nchar(x))
-}
 
 ##' upscale_met upscales the temporal resolution of a dataset
 ##' @name upscale_met
 ##' @title upscale_met
 ##' @export
-##' @param outfolder
-##' @param input_met - met dataset
-##' @param site.id
-##' @param reso - numeric value in HOURS that is the desired resolution for the dataset to be upscaled to
-##'               # might need to change this for monthly/yearly option later
-##' @param overwrite
-##' @param verbose
-##' @author James Simkins
+##' @param input_met path to netcdf file containing met dataset
+##' @param outfolder path to directory where output should be saved
+##'           Output is netcdf with named as <input_met_filename>.upscaled.nc
+##' @param resolution numeric value that is the desired resolution for the dataset to be upscaled to
```

Comment:
@infotroph even if the input met isn't days, couldn't the `resolution` argument could be defined as a numeric value with units of days? Inside the function, you could convert the input_met time series to units of days. (or otherwise the rescale the value of `resolution` to match the input. 

My only concern is that we minimize the number of arguments and resulting cognitive load.